### DISCUSSION TITLE
Welcome to neominimap.nvim Discussions!

### DISCUSSION THREAD

**Discussion by Isrothy** [Isrothy/neominimap.nvim#115] [Announcements]
# Welcome to Neominimap.nvim Discussions!

Welcome to the Neominimap.nvim discussions! 🎉

This is the place to discuss all things related to Neominimap.nvim. Whether you're a new user looking for help, a developer interested in contributing, or simply someone who wants to share ideas and feedback, you're in the right place.

## What to Discuss Here:

- **General Questions**: If you're unsure about how to use a specific feature or have any questions about the plugin, feel free to ask.
- **Tips and Tricks**: Have you found a cool way to use Neominimap.nvim or integrate it with other plugins? Share your tips with the community!
- **Plugin Development**: Interested in contributing to Neominimap.nvim? Let’s discuss how we can collaborate to make the plugin even better.
- **General Feedback**: Share your thoughts on the plugin, and let us know how it's working for you.

## Where to Report Issues:

- **Feature Requests**: If you have an idea for a new feature, please open an issue [here](https://github.com/Isrothy/neominimap.nvim/issues) to start the discussion.
- **Bug Reports**: Encountered a problem? Please report it in the [issues section](https://github.com/Isrothy/neominimap.nvim/issues) with as much detail as possible so we can work on a fix.

## How to Get Started:

1. **Introduce Yourself**: Start by saying hello and letting us know how you're using Neominimap.nvim.
2. **Check Existing Topics**: Before starting a new discussion, please check if there’s already a topic on the subject.
3. **Be Respectful**: This is a community space, so please be respectful and considerate of others.

We’re excited to have you here and look forward to your contributions!

Happy coding! 👩‍💻👨‍💻

**Comment by xdanep** [comment]
Good morning, I'm configuring neominimap but I don't know how to remove the dots that represents empty space. Very nice plugin by the way.
![image](https://github.com/user-attachments/assets/8fc08909-d39e-4a68-9b67-3be85377b122)

**Reply by Isrothy** [reply]
I guess this issue is related to the font. Have you tried using another font?

**Reply by xdanep** [reply]
I tried with JetBrains Nerd Font and Hack Nerd Font, maybe the issue is related to the terminal? i'm using alacritty

**Reply by Isrothy** [reply]
I guess what you have to do is configuring alacritty to use suitable icons.

**Reply by xdanep** [reply]
Alacritty was the problem, I changed to kitty, thanks for your help

**Reply by Isrothy** [reply]
It doesn't seem that this issue is related to your Neovim configuration. It appears to be more related to the terminal emulators and fonts. You might want to look into settings that allow for proper display of `⠸` and `⠹`.

**Comment by llanosrocas** [comment]
Hello.
I just want to thank you for this project. This is exactly what I've been looking for: zero usage problems, good documentation (which is rare), and many customization options. I think this project is really awesome and a great example for other developers to learn from.

**Reply by Isrothy** [reply]
Thanks so much for your kind words! It means a lot to me.
